apps/src/gamelab/GameLab.js

```diff
@@ -88,8 +88,17 @@ var GameLab = function () {
   consoleApi.setLogMethod(this.log.bind(this));
   errorHandler.setLogMethod(this.log.bind(this));
 
-  /** Expose for testing **/
-  window.__mostRecentGameLabInstance = this;
+  if (process.env.NODE_ENV !== "production") {
```

Comment:
Do we want to expose this on production or only levelbuilder? Where should they be getting links for these animations?